Assets/Scripts/Tooltip/TooltipDisplay.cs

```diff
@@ -158,8 +158,8 @@ public void UpdateTooltip(TooltipType type)
                     details.text = $"{Manager.Stats.Stability}/100 town stability ({(change > 0 ? "+" : "") + change} next turn)";
                     break;
                 case Stat.Spending:
-                    details.text = $"{Manager.Stats.WealthPerTurn} wealth per turn from {Manager.Adventurers.Count} adventurers \n" +
-                                   $"(5 wealth per adventurer) times {(100 + Manager.Stats.GetStat(Stat.Spending))/100f} from spending modifier.\n" +
+                    details.text = $"{Manager.Stats.WealthPerTurn} wealth per turn\n" +
+                                   $"5 per adventurer({Manager.Adventurers.Available}) times {(100 + Manager.Stats.GetStat(Stat.Spending))/100f} spending modifier.\n" +
```

Comment:
I would avoid using literals in UI displaying stuff in case there's balancing changes made in future, change these to constants if possible.